sklearn/decomposition/_nmf.py

```diff
@@ -982,6 +1110,13 @@ def non_negative_factorization(X, W=None, H=None, n_components=None, *,
     shuffle : bool, default=False
         If true, randomize the order of coordinates in the CD solver.
 
+    forget_factor : float, default=None.
+        Amount of rescaling of past information. Its value is 1 for batch
+        NMF algorithm, it could be <1 for online NMF algorithm. Only for
+        MiniBatch implementation.
```

Comment:
I find it a bit unclear. You say that 1 is for batch NMF but also that it's only for MiniBatchNMF. I'd remove the part about batch nmf and just say that it's only used for minibatch nmf